data/common.yaml

```diff
@@ -1 +1,16 @@
---- {}
+---
+
+profile_gpu::dcgm::install::bind_mount_install:        # See reference.md for when this is needed, set to undef on purpose
```

Comment:
Suggest explicitly setting bind_mount_install to null (Hiera/YAML null corresponds to Puppet's undef). But would need to check and make sure this still fails.

Reply:
Discussed that is not necessary because we're not relocating, we're using the bind mount.